src/bootstrap/compile.rs

```diff
@@ -1106,9 +1106,7 @@ impl Step for Sysroot {
         let compiler = self.compiler;
         let host_dir = builder.out.join(&compiler.host.triple);
         let sysroot = if compiler.stage == 0 {
-            host_dir.join("stage0-sysroot")
-        } else if builder.download_rustc() {
-            host_dir.join("ci-rustc-sysroot")
```

Comment:
It's probably best to not delete these lines, though @jyn514 knows better.

Reply:
ok @compiler-errors will add that line back to the code.

Reply:
can one of  you @compiler-errors  @jyn514 help me with the below thing 

warning: there have been changes to x.py since you last updated.
help: consider looking at the changes in `src/bootstrap/CHANGELOG.md`
note: to silence this warning, update `config.toml` to use `changelog-seen = 3` instead
downloading https://static.rust-lang.org/dist/2022-09-20/rustfmt-nightly-aarch64-apple-darwin.tar.xz
############################################################################################################################################################################################################### 100.0%
extracting /Users/chetankokil/git/rust/build/cache/2022-09-20/rustfmt-nightly-aarch64-apple-darwin.tar.xz to /Users/chetankokil/git/rust/build/aarch64-apple-darwin/bootstrap-sysroot


failed to execute command: "/Users/chetankokil/git/rust/build/aarch64-apple-darwin/bootstrap-sysroot/bin/rustc" "--print" "sysroot"
error: No such file or directory (os error 2)
```.

This is not allowing me to build or push

Reply:
Yes, the `download_rustc()` branch should not be modified at all.